### PR TITLE
pmap: add page

### PR DESCRIPTION
**Pull request by AshishShenoy**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/pmap.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+> Report memory map of a process or processes.
+> More information: <https://manned.org/pmap>.
+
+- Print memory map for a process selected by its process id (PID):
+
+`pmap {{pid}}`
+
+- Show the extended format:
+
+`pmap -x {{pid}}`
```
Comment (navarroaxel) [inline pages/linux/pmap.md:12]:
```suggestion
`pmap --extended {{pid}}`
```

File: pages/linux/pmap.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Show the device format:
+
+`pmap -d {{pid}}`
```
Comment (navarroaxel) [inline pages/linux/pmap.md:16]:
```suggestion
`pmap --device {{pid}}`
```

File: pages/linux/pmap.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Limit results to a memory address range specified by _low_ and _high_:
+
+`pmap -A {{low}},{{high}}`
```
Comment (navarroaxel) [inline pages/linux/pmap.md:20]:
```suggestion
`pmap --range {{low}},{{high}}`
```

File: pages/linux/pmap.md (hunk 2 above)
Comment (mfrw) [inline pages/linux/pmap.md:16]:
```suggestion
`pmap --device {{pid}}`
```

File: pages/linux/pmap.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# pmap
+
+> Report memory map of a process or processes.
```
Comment (navarroaxel) [inline pages/linux/pmap.md:3]: But the command applies to one process at time.
```suggestion
> Report memory map of a process.
```

Comment (AshishShenoy) [reply pages/linux/pmap.md:3]: The command can be used on more processes as such:

`pmap {{pid1}} {{pid2}}`

However I will commit this suggestion as it is not a common use case.

Comment (navarroaxel) [reply pages/linux/pmap.md]: add an example for this.

Comment (AshishShenoy) [reply pages/linux/pmap.md:3]: Done

File: pages/linux/pmap.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Show the extended format:
+
+`pmap --extended {{pid}}`
+
+- Show the device format:
+
+`pmap --device {{pid}}`
+
+- Limit results to a memory address range specified by _low_ and _high_:
```
Comment (EmilySeville7cfg) [inline pages/linux/pmap.md:18]:
```suggestion
- Limit results to a memory address range specified by `low` and `high`:
```

File: pages/linux/pmap.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+`pmap --range {{low}},{{high}}`
+
+- Print memory maps for multiple processes:
+
+`pmap {{pid1}} {{pid2}}`
```
Comment (EmilySeville7cfg) [inline pages/linux/pmap.md:24]:
```suggestion
`pmap {{pid1 pid2 ...}}`
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

**Review by EmilySeville7cfg — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**